lib/config/migrations/custom/gradle-lite-migration.ts

```diff
@@ -0,0 +1,18 @@
+import is from '@sindresorhus/is';
+import { mergeChildConfig } from '../../utils';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class GradleLiteMigration extends AbstractMigration {
+  override readonly deprecated = true;
+  override readonly propertyName = 'gradle-lite';
+
+  override run(value: unknown): void {
+    const gradle = this.get('gradle');
+
+    if (is.nonEmptyObject(value)) {
+      const newGradle = mergeChildConfig(gradle ?? {}, value);
```

Comment:
You probably need to cast `value`

Reply:
Yes, it helped, but now build failed and some strange issues with code coverage.